data-raw/R2R_baseline_habitat_inputs/cache_baseline_hab.R

```diff
@@ -288,9 +289,484 @@ fp |>
   facet_wrap(~watershed, scales = 'free_y') + 
   theme_minimal()
 
+# SPRING RUN ---------------------------------------------------------------------
+# Update DSMhabitat values 
+# set r_to_r_baseline_fr_spawn to 
+r_to_r_baseline_sr_spawn <- DSMhabitat::sr_spawn$biop_itp_2018_2019
+
+# Add american river spawning habitat 
+add_project_habitat <- DSMhabitat::sr_spawn$biop_itp_2018_2019["Cottonwood Creek", , ] * 
+  hab_prop_change_from_projects("spawning", "Cottonwood Creek", "sr", "adult", "biop_itp_2018_2019")
+updated_habitat <- DSMhabitat::sr_spawn$biop_itp_2018_2019["Cottonwood Creek", , ] + add_project_habitat
+
+r_to_r_baseline_sr_spawn["Cottonwood Creek", , ] <- updated_habitat 
+
+# add clear creek spawning habitat
```

Comment:
comment says Clear but you added Deer

Reply:
Updated, and checked/updated comments throughout